src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -0,0 +1,263 @@
+        }
+
+        static AnnotationManager ofAll(final List<String> annotationsToExclude){
+            final List<String> groups = getAllAnnotationGroupNames();
+            final List<String> annots = getAllAnnotationNames();
+            annots.removeAll(annotationsToExclude);
+            return new AnnotationManager(groups, annots);
+        }
+
+        private static List<String> getAllAnnotationNames() {
+            final Set<VariantAnnotation> union = Sets.union(new HashSet<>(makeAllGenotypeAnnotations()), new HashSet<>(AnnotationManager.makeAllInfoFieldAnnotations()));
+            return union.stream().map(a -> a.getClass().getSimpleName()).collect(Collectors.toList());
+        }
+
+        public static List<String> getAllAnnotationGroupNames() {
+            return Stream.of(AnnotationGroup.values()).map(v -> v.name()).collect(Collectors.toList());
+        }
+
+        public List<InfoFieldAnnotation> createInfoFieldAnnotations() {
+            final List<InfoFieldAnnotation> all = makeAllInfoFieldAnnotations();
+            return filterAnnotations(all);
+        }
+
+        private static List<InfoFieldAnnotation> makeAllInfoFieldAnnotations() {
+            return Arrays.asList(
+                    new InbreedingCoeff(Standard, ActiveRegionBased),
+                    new Coverage(Standard, ActiveRegionBased),
+                    new ReadPosRankSumTest(Standard, ActiveRegionBased),
+                    new BaseQualityRankSumTest(Standard, ActiveRegionBased),
+                    new MappingQualityRankSumTest(Standard, ActiveRegionBased),
+                    new ChromosomeCounts(Standard, ActiveRegionBased),
+                    new StrandOddsRatio(Standard, ActiveRegionBased),
+                    new FisherStrand(Standard, ActiveRegionBased),
+                    new RMSMappingQuality(Standard, ActiveRegionBased),
+                    new QualByDepth(Standard, ActiveRegionBased),
+                    new MappingQualityZero(ActiveRegionBased),
+                    new SampleList(),
+                    new ClippingRankSumTest(ActiveRegionBased),
+                    new LikelihoodRankSumTest(ActiveRegionBased),
+                    new TandemRepeat(ActiveRegionBased),
+                    new GenotypeSummaries(ActiveRegionBased)
+                    //        new PossibleDeNovo(),  //TODO enable this
+            );
+        }
+
+        public List<GenotypeAnnotation> createGenotypeAnnotations() {
+            final List<GenotypeAnnotation> all = makeAllGenotypeAnnotations();
+            return filterAnnotations(all);
+        }
+
+        private static List<GenotypeAnnotation> makeAllGenotypeAnnotations() {
+            return Arrays.asList(
+                    new BaseQualitySumPerAlleleBySample(Standard, ActiveRegionBased),
+                    new DepthPerAlleleBySample(Standard, ActiveRegionBased),
+                    new OxoGReadCounts(ActiveRegionBased)
```

Comment:
to be clear - this will still hardwire the classes loaded to those defined in a given package.